www/js/controllers/database-controller.js

```diff
@@ -0,0 +1,14 @@
+module.controller('MyCtrl', function($scope, $cordovaSQLite) {
```

Comment:
Thanks, Brian! I corrected the controller name. 

Eric

> On Mar 30, 2016, at 3:42 PM, Brian Gregg <notifications@github.com> wrote:
> 
> In www/js/controllers/database-controller.js:
> 
> > @@ -0,0 +1,14 @@
> > +module.controller('MyCtrl', function($scope, $cordovaSQLite) {
> Cool, just make sure that you name the controller something that matches the name of the file. So I would name it 'DatabaseController' as opposed to 'MyCtrl'
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>